src/main/java/frc/robot/subsystems/vision/Camera.java

```diff
@@ -24,40 +24,53 @@
 import edu.wpi.first.math.util.Units;
 
 public class Camera {
-
-  public final String name;
-  public final String path;
-  public final Transform3d transform;
-  public final CameraIntrinsics intrinsics;
-
-  public final PhotonCamera camera;
-  public final PhotonPoseEstimator estimator;
-
-  private final double MAX_PITCHROLL = Units.degreesToRadians(5);
-  private final double MAX_Z = Units.inchesToMeters(7);
-
-  private final double xyStdDevCoefficient = 0.08;
-  private final double thetaStdDevCoefficient = 0.16;
-  private final double coefficientFactor = 10_000.0;
+  private String path;
+  private Transform3d transform;
+  private CameraIntrinsics intrinsics;
+  private AprilTagFieldLayout field;
+  private PhotonCamera camera;
+  
+  private final PhotonPoseEstimator estimator;
+  
+  private final double MAX_PITCHROLL_DEGREES = 5;
+  private final double MAX_PITCHROLL = Units.degreesToRadians(MAX_PITCHROLL_DEGREES);
```

Comment:
a lot of constants would be pretty long if you did this - I personally wouldn't do that as I feel the semantic meaning is still clear as it was originally. 

That being said, most of these, except the path names, should be moved to `VisionConstants.java`.